[BUG]: Project import/export seems to be broken

Nelson reported exporting a project from Production into Staging, but the copy/import process seemed to be incomplete.

There are numerous errors of these type in the server logs that seem to indicate we didn't copy over the dataset assets correctly:

```
2025-03-11 16:15:56 [ERROR] Unable to get dataset [software.uncharted.terarium.hmiserver.controller.dataservice.DatasetController:184]
org.springframework.web.server.ResponseStatusException: 404 NOT_FOUND "We couldn't find the requested dataset. Please verify that the dataset you selected is correct or contact support."
```

[Demo Project (Hepatitis A).zip](https://github.com/user-attachments/files/19229631/Demo.Project.Hepatitis.A.zip)
[Demo Project (Zombies).zip](https://github.com/user-attachments/files/19229632/Demo.Project.Zombies.zip)

Projects from Nelson:


Looks like fetching Dataset is the main cause based on @mwdchang review

--- a/packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelConfigurationController.java
+++ b/packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelConfigurationController.java
@@ -292,7 +292,7 @@ public class ModelConfigurationController {
 	@GetMapping("/{id}/original-model")
 	@Secured(Roles.USER)
 	@Operation(summary = "Get the original model of which the configuration was created for")
-	@HasProjectAccess(level = software.uncharted.terarium.hmiserver.utils.rebac.Schema.Permission.WRITE)
+	@HasProjectAccess
 	@ApiResponses(
 		value = {
 			@ApiResponse(

--- a/packages/server/src/test/resources/application-test.properties
+++ b/packages/server/src/test/resources/application-test.properties
@@ -140,6 +140,9 @@ terarium.taskrunner.rabbitmq.table_extraction.password=terarium123
 terarium.taskrunner.rabbitmq.text_extraction.addresses=amqp://localhost:5672
 terarium.taskrunner.rabbitmq.text_extraction.username=terarium
 terarium.taskrunner.rabbitmq.text_extraction.password=terarium123
+terarium.taskrunner.rabbitmq.ocr_extraction.addresses=amqp://localhost:5672
+terarium.taskrunner.rabbitmq.ocr_extraction.username=terarium
+terarium.taskrunner.rabbitmq.ocr_extraction.password=terarium123
 
 ########################################################################################################################
 # Logging

